src/Connections.Abstractions/breakingchanges.netcore.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Connections.DefaultConnectionContext : Microsoft.AspNetCore.Connections.ConnectionContext, System.IDisposable, Microsoft.AspNetCore.Connections.Features.IConnectionIdFeature, Microsoft.AspNetCore.Connections.Features.IConnectionItemsFeature, Microsoft.AspNetCore.Connections.Features.IConnectionTransportFeature, Microsoft.AspNetCore.Connections.Features.IConnectionUserFeature, Microsoft.AspNetCore.Connections.Features.IConnectionLifetimeFeature",
```

Comment:
Adding this exception since we moved the Abort() method in https://github.com/aspnet/KestrelHttpServer/commit/ac31e5ab30e6f6023ddb12b24f76fa72e764a94a#diff-499c3a0185b3217a93e225bb80161218 but the API check tool can't figure out the method has been relocated and still exist on this type.

Reply:
Hmm, that's technically a binary break. Putting Abort back on the child class and having it call the base class would be safer. Try it?

Reply:
Moving a method from a derived class to a base class is not a breaking change at all, is it?

Reply:
According to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#members:

> Members
> ✓ Allowed
> ...
> Moving a member onto a class higher in the hierarchy tree of the type from which it was removed

Reply:
Ok, can we update the tool so we don't have this exception then?

Reply:
Filed https://github.com/aspnet/BuildTools/issues/767. However, I'm going to go with the breaking changes exception for now since I don't want to wait until that's resolved before checking this in.

Reply:
Apparently we are moving from ApiCheck to ApiCompat, so not sure if we'll update ApiCheck.

Reply:
That's right, there is no planned work on the tool.